IOPool/Common/test/TestEdmFastMerge.sh

```diff
@@ -2,74 +2,71 @@
 
 #---------------------------
 # Create files with different ancestors
 #---------------------------
 
-cmsRun --parameter-set ${LOCAL_TEST_DIR}/PreFastMergeTest_ancestor1_cfg.py || die 'Failure using PreFastMergeTest_ancestor1_cfg.py' $?
-cmsRun --parameter-set ${LOCAL_TEST_DIR}/PreFastMergeTest_ancestor2_cfg.py || die 'Failure using PreFastMergeTest_ancestor2_cfg.py' $?
+cmsRun ${LOCAL_TEST_DIR}/PreFastMergeTest_ancestor1_cfg.py || die 'Failure using PreFastMergeTest_ancestor1_cfg.py' $?
+cmsRun ${LOCAL_TEST_DIR}/PreFastMergeTest_ancestor2_cfg.py || die 'Failure using PreFastMergeTest_ancestor2_cfg.py' $?
 
 
 #---------------------------
 # Merge files
 #---------------------------
-cmsRun -j ${LOCAL_TMP_DIR}/TestFastMergeFJR_ancestor.xml --parameter-set ${LOCAL_TEST_DIR}/FastMergeTest_ancestor_cfg.py || die 'Failure using FastMergeTest_ancestor_cfg.py' $?
-cmsRun -j ${LOCAL_TMP_DIR}/TestFastMergeFJR_ancestor2.xml --parameter-set ${LOCAL_TEST_DIR}/FastMergeTest_ancestor2_cfg.py || die 'Failure using FastMergeTest_ancestor2_cfg.py' $?
+cmsRun -j ${LOCAL_TMP_DIR}/TestFastMergeFJR_ancestor.xml ${LOCAL_TEST_DIR}/FastMergeTest_ancestor_cfg.py || die 'Failure using FastMergeTest_ancestor_cfg.py' $?
+cmsRun -j ${LOCAL_TMP_DIR}/TestFastMergeFJR_ancestor2.xml ${LOCAL_TEST_DIR}/FastMergeTest_ancestor2_cfg.py || die 'Failure using FastMergeTest_ancestor2_cfg.py' $?
 
 
-cmsRun -j ${LOCAL_TMP_DIR}/TestFastMergeFJRx.xml --parameter-set ${LOCAL_TEST_DIR}/FastMergeTest_x_cfg.py || die 'Failure using FastMergeTest_x_cfg.py' $?
+cmsRun -j ${LOCAL_TMP_DIR}/TestFastMergeFJRx.xml ${LOCAL_TEST_DIR}/FastMergeTest_x_cfg.py || die 'Failure using FastMergeTest_x_cfg.py' $?
 #need to filter items in job report which always change
 egrep -v "<GUID>|<PFN>|^$" $LOCAL_TEST_DIR/proper_fjrx_output > $LOCAL_TMP_DIR/proper_fjrx_output_filtered
 egrep -v "<GUID>|<PFN>|^$" $LOCAL_TMP_DIR/TestFastMergeFJRx.xml  > $LOCAL_TMP_DIR/TestFastMergeFJRx_filtered.xml
 diff $LOCAL_TMP_DIR/proper_fjrx_output_filtered $LOCAL_TMP_DIR/TestFastMergeFJRx_filtered.xml || die 'output framework job report is wrong for proper_fjrx_output_filtered' $?
 
-#cmsRun -j ${LOCAL_TMP_DIR}/TestFastMergeFJRx_second.xml --parameter-set ${LOCAL_TEST_DIR}/FastMergeTest_x_second_cfg.py || die 'Failure using FastMergeTest_x_second_cfg.py' $?
+#cmsRun -j ${LOCAL_TMP_DIR}/TestFastMergeFJRx_second.xml ${LOCAL_TEST_DIR}/FastMergeTest_x_second_cfg.py || die 'Failure using FastMergeTest_x_second_cfg.py' $?
 ##need to filter items in job report which always change
 #egrep -v "<GUID>|<PFN>|^$" $LOCAL_TEST_DIR/proper_fjrx_second_output > $LOCAL_TMP_DIR/proper_fjrx_second_output_filtered
 #egrep -v "<GUID>|<PFN>|^$" $LOCAL_TMP_DIR/TestFastMergeFJRx_second.xml  > $LOCAL_TMP_DIR/TestFastMergeFJRx_second_filtered.xml
 #diff $LOCAL_TMP_DIR/proper_fjrx_second_output_filtered $LOCAL_TMP_DIR/TestFastMergeFJRx_second_filtered.xml || die 'output framework job report is wrong for proper_fjrx_second_output_filtered' $?
```

Comment:
I wonder if this commented out test should be enabled (possibly in a later PR), or removed. I see it was added (as commented out) in 5f1487b85d0e.

Reply:
@Dr15Jones what do you suggest here? 
@makortel , changes for this can go in a separate PR